### PR TITLE
Add the channel Every Steve Jobs's Video

### PR DESCRIPTION
**Pull request by marciogomes**
Added the channel ["Every Steve Jobs's Video"](https://www.youtube.com/user/EverySteveJobsVideo/playlists), with a list of notable videos.

### CODE REVIEW

**Comment by davidsonfellipe** [comment]
@marciogomes What amazing playlist!

**Comment by marciogomes** [comment]
Thanks, @davidsonfellipe!

**Comment by watson** [comment]
@marciogomes Thanks :smiley: There's a lot of material there!

I'm a big fan of Steve Jobs my self, but I've been reluctant to include this channel because it's mostly interesting for people interested in Steve Jobs. The channel as a whole isn't about computer history in general. But there's a lot of videos in there that *is* relevant for computer history - just like the Macintosh introduction that you also linked to.

I've also received a lot of submissions for videos about hacker culture which I don't really think fit into the main list - but have so far accepted them anyway. But I might make a sub-section with hacker culture videos and move them into there - anyway, that's another topic :wink: - Maybe we should make sub-sections for notable people that have had a huge impact on computer history. Then we can put this channel in there... What do you think about that?

File: README.md [hunk 1]
```diff
@@ -27,6 +27,9 @@ A curated list of computer history videos, documentaries and related folklore ma
 - [Xerox Parc - Office Alto Commercial](https://www.youtube.com/watch?v=M0zgj2p7Ww4) (1972)
 - [The UNIX System: Making Computers More Productive](https://www.youtube.com/watch?v=tc4ROCJYbm0) (1982)
 - [The UNIX System: Making Computers Easier to Use](https://www.youtube.com/watch?v=XvDZLjaCJuw) (1982)
+- [Every Steve Jobs Video](https://www.youtube.com/user/EverySteveJobsVideo/playlists) (1980-2011) - Steve Jobs's largest video archive. A few of the notable videos are:
```
Comment (watson) [inline README.md]: As far as I can see the playlist goes back to 1978

**Comment by marciogomes** [comment]
I'm totally agree with you, @watson. If you want, I can create this subsection of notable people and include this channel.

**Comment by watson** [comment]
@marciogomes please do :smiley:

**Pull request closed without merging**